tests/Makefile

```diff
@@ -102,6 +102,15 @@ DATA_FILES := $(wildcard suites/test_suite_*.data)
 DATA_FILES += $(filter-out $(DATA_FILES),$(GENERATED_FILES))
 APPS = $(basename $(subst suites/,,$(DATA_FILES)))
 
+# psasim takes an extremely long execution time on some test suites so we
+# exclude them from the list.
+ifdef PSASIM
+PSASIM_EXCLUDE = suites/test_suite_constant_time_hmac.data \
+				suites/test_suite_lmots.data \
+				suites/test_suite_lms.data
+DATA_FILES := $(filter-out $(PSASIM_EXCLUDE), $(DATA_FILES))
+endif
+
```

Comment:
Why are we adding a new way to exclude test cases? Why are they excluded from being built, instead of using `env SKIP_TEST_SUITE=constant_time_hmac,lmots,lms make test` to skip them at runtime?

Reply:
They are excluded from being built as until recently we couldn't build the whole set of tests with `PAS_CRYPTO_CLIENT`

Reply:
> Why are we adding a new way to exclude test cases? Why are they excluded from being built, instead of using env SKIP_TEST_SUITE=constant_time_hmac,lmots,lms make test to skip them at runtime?

I completely missed this feature, sorry. I will update the PR to follow your suggestion.